src/ADMIN/components/AdminHeader.jsx

```diff
@@ -50,25 +50,25 @@ function AdminHeader() {
           </Link>
           <Link
             className="text-[#7E8180] hover:text-[#009975] transition-all duration-300 cursor-pointer"
-            to="/blogs"
+            to="/admin"
           >
             Blogs
           </Link>
           <Link
             className="text-[#7E8180] hover:text-[#009975] transition-all duration-300 cursor-pointer"
-            to="/chapters"
+            to="/admin"
           >
             Chapters
```

Comment:
Conflict on this line 
```sql
<<<<<<< main
            to="/admin"
=======
            to="/admin/all-chapters"
>>>>>>> main```

whichone is the correct ref?

Reply:
It should be `to="/admin/all-chapters"` for _Chapters_.